en/guide/configuration_file.md

```diff
@@ -109,18 +109,54 @@ Example (Ubuntu):
 video0=http://127.0.0.1:8000/camera-def-uvc.xml
 ```
 
-### [imgcap]
+### [imgcap] {#imgcap}
+
+This section defines the *default* values used for image capture. With the exception of `location` it should be possible override the values over MAVLink using parameters defined in the [Camera Definition File](../guide/camera_definition_file.md).
+
+> **Note** At time of writing these default values cannot yet be overridden (see [#161](https://github.com/Dronecode/camera-streaming-daemon/issues/161)).
 
 Key | Description | Default
 -- | --- | ---
-`location` | The local path with write permission where captured images will be saved (usually the system-standard "Temp" directory) | -
+`width` | Width of the image to be captured in pixels. | Full width of camera frame for sensor type (i.e 1080P - 1920, 720P - 1280, etc.)
+`height` | Height of the image to be captured in pixels. | Full height of camera frame for sensor type (i.e 1080P - 1080, 720P - 720, etc).
+`format` | Image format (number). <br>Possible values:<ul><li>2 (Jpeg/<code>IMAGE_FILE_JPEG</code>)</li></ul>Notes:<ul><li>Currently only Jpeg is <a href="https://github.com/Dronecode/camera-streaming-daemon/issues/163">supported for image capture</a>.</li></ul> | 2 (JPEG).
+`location` | The local path with write permission where captured images will be saved (usually the system-standard "Temp" directory). The path should be accessible and writeable. | /tmp/
 
 Example:
 ```
 [imgcap]
 location=~/Temp/
```

Comment:
Example for other fields  can be added

Reply:
Thanks, done! https://github.com/Dronecode/camera-manager-docs/commit/b07e19d182f0758431a19689371bf5c42f5aa66b